### PR TITLE
Adds businessRespondentStatus to associations

### PR DESCRIPTION
**Pull request by jwgwalton**

### CODE REVIEW

**Review by gemmai95 — CHANGES_REQUESTED** [review]
This should probably have tests around it.

File: ras_party/models/models.py [hunk 1]
```diff
@@ -213,7 +214,8 @@ def _get_business_associations(businesses):
         for business_respondent in businesses:
             business_dict = {
                 "partyId": business_respondent.business.party_uuid,
-                "sampleUnitRef": business_respondent.business.business_ref
+                "sampleUnitRef": business_respondent.business.business_ref,
+                "businessRespondentStatus": business_respondent.status.name
```
Comment (tej99) [inline ras_party/models/models.py]: Missing respondent:
"businessRespondentStatus": business_respondent.**respondent**.status.name

**Review by tej99 — CHANGES_REQUESTED** [review]

File: ras_party/models/models.py (hunk 1 above)
Comment (jwgwalton) [reply ras_party/models/models.py]: Ooh that might be why it broke.

**Review by gemmai95 — COMMENTED** [review]
Have you updated the API.md to reflect this now being returned?

**Review by tej99 — APPROVED** [review]

**Comment by jwgwalton** [comment]
@gemmai95 Added the changes to API.md

**Review by gemmai95 — APPROVED** [review]

**Pull request merged**